tests/python/relay/utils/adreno_utils.py

```diff
@@ -116,3 +117,14 @@ def gpu_preprocess(tvm_mod):
             mod = tvm.IRModule.from_expr(tvm_mod)
             tvm_mod_nchwc = seq(mod)
             return tvm_mod_nchwc
+
+
+def gpu_preprocess_nhwc(tvm_mod):
```

Comment:
we do not use this func, right? is it artifact of some intermediate test implementations?
Need to remove?